pkgs/development/python-modules/pybind11/default.nix

```diff
@@ -30,6 +39,7 @@ buildPythonPackage rec {
 
   nativeBuildInputs = [ cmake ];
   buildInputs = lib.optionals (pythonOlder "3.9") [ libxcrypt ];
+  propagatedBuildInputs = [ setupHook ];
```

Comment:
Maybe rename this to pybindCmakeSetupHook to be more specific?

Reply:
I renamed the function in the setup hook and added a comment.